website/archiver/utils.py

```diff
@@ -322,7 +329,7 @@ def _make_file_response(file_info, parent_guid):
     archived_file_id = file_info['path'].lstrip('/')
     return {
         'file_id': archived_file_id,
-        'file_name': bleach.clean(file_info['name']).replace('&amp;', '&'),
+        'file_name': normalize_unicode_filename(bleach.clean(file_info['name']).replace('&amp;', '&'))[0],
```

Comment:
Doesn't `normalize_unicode_filename` already replace the `&amp;` with an `&`, or is there double-encoding happening here?